cmd/capibmadm/options/options.go

```diff
@@ -28,11 +32,32 @@ var GlobalOptions = &options{}
 type options struct {
 	IBMCloudAPIKey    string
 	ServiceInstanceID string
+	PowerVSZone       string
+	VPCRegion         string
+	ResourceGroupName string
+	Debug             bool
+	Output            printer.PType
+}
+
+// AddCommonFlags will add common flags to the cli.
+func AddCommonFlags(cmd *cobra.Command) {
+	cmd.Flags().Var(&GlobalOptions.Output, "output", "Supported printer types: table, json")
```

Comment:
can we set `PrinterTypeTable` as a default value?

Reply:
set explicitly. `cmd.Flags().Var()` not providing a option set default val

Reply:
cmd.Flags().VarP() should give you an option to do so..

Reply:
ignore!